packages/dashboard/src/components/Wizard/PanelWizard.types.ts

```diff
@@ -0,0 +1,24 @@
+import { IStyle, ITheme } from 'office-ui-fabric-react/lib/Styling';
+import { IWizardProps } from './Wizard.types';
+import { IPanelProps } from 'office-ui-fabric-react/lib/Panel';
+
+export interface IPanelWizardProps {
```

Comment:
>IPanelWizardProps [](start = 17, length = 17)

Is it better to derive from "IWizardProps" than to include as property?

Reply:
I can't, since `IWizardProps` contains property `styles` and `IPanelWizardProps` also contains property `styles`